QUARKUS-1415.md

```diff
@@ -0,0 +1,28 @@
+# QUARKUS-1415 Reactive REST Client improvements
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1415
+
+Test plan covers RESTEasy Reactive functionality provided by `quarkus-resteasy-reactive` and `quarkus-rest-client-reactive` extensions,
+with emphasis on multipart form data support.
+
+https://quarkus.io/version/main/guides/resteasy-reactive
+
+https://quarkus.io/version/main/guides/rest-client-reactive
+
+## Related test plans
+- QUARKUS-1075
+
+## Scope of the testing
```

Comment:
These topics are not covered:
 - support for ParamConverter
 - recursive sub-resources support, that is sub-sub-resources and so on work too,
 - UI / dev experience - List of clients in Dev UI and Links to client interfaces (if in user's app)

https://resteasy.dev/2021/04/05/blog-ParamConverter-with-Quarkus/ can be used for ParamConverter coverage

sub-resources can be a bit tricky topic, we should make sure one can't end up with infinite loop of sub-resources

UI/dev experience should be checked as part of the initial experiments, not necessary to de TD for it, but should be checked at least manually though